### ISSUE TITLE
Core Team Meeting IV Retro — agenda 

### ISSUE THREAD

**Issue by Malwine** [clojurebridge-berlin/organization#86]
## Retrospective Meeting (July 5, 2016)

**Attending:** Arne, Ben, Franka, Julia, Malwine and Nicola
**Format:** In the beginning everybody wrote down the 3 most important topics for themselves and then we discussed most of them.

🅰️ = action point
### Food
- lesson learned: people are really hungry at the workshop
- we need more food
- maybe a simpler
- 🅰️ Discuss dinner before after party or venue with dinner option.
### After party
- good: lots of space, accessible, affordable
- outside was ok, inside would have been too dark
- bad: no food
### Accessible quite room
- problem: the quite room was not accessible during talks
- 🅰️ Find accessible quite room
### Venue
- small rooms are nice
- lack of accessible bathrooms is bad
- high security building possibly makes people feel uncomfortable, on the other hand doorkeeper
- we could have 10-20 more people at Wunderlist
- 85 people? 25 extra next time?
- possible new venues: Babbel, ImmobilienScout, Microsoft - unter den Linden, HTW Schöneweide 
- keep an eye open for new venues
- 🅰️ venue should be decided before organization for the next workshop starts 
### Space management
- having orange lanyards was good
- Important question at the workshop: who is there and why?
- 🅰️  visualize role on the badge: coach, attendee, volunteer
- 🅰️ have a list of who is inside the building for emergency
### Guest policy
- possible guest personas: strangers, applicants who didn't get selected, press, friends, (volunteers)
- generally we would like to avoid having last minute guests, we need to have a policy
- there are 2 kinds of guests, invited and uninvited
  - we wish to introduce **invited** guests at one of our presentations e.g. "This is {name}, she works for {sponsor/foundation/press} and is here today to see the workshop"
  - for **uninvited** guest who just drop by last minute we should have a short Q&A:
    - Who are you? How are you connected to ClojureBridge? How many people are with you?
    - based on the answers of this questions at least 3 orga people decide if they let these guests in
### Orga communication
- Experience: there are situations where you have to take a hard decision, sometimes on your own and sometimes they were not communicated well
- new process: 
  - given you have a decision to make
  - find 3 other organizers or call them to get where you are
  - decide together
  - broadcast and inform via Slack
- 🅰️ Share a list of all organizers phone numbers before the event
- 🅰️ Have a on-call-person
### Organizer collaboration
- continue: having meeting notes
- problems/ problematic questions: 
  - account details sharing
  - Who is responsible for what?
  - Who decides what?
- We want to include new people in a better and engaging way, they need to know where to start
- We need to keep people updated, maybe with a newsletter?
- We might need a person that delegates tasks to orgas and volunteers?
- Trello is good for orgas to keep track but not for volunteers, how to keep people in the loop?
- Slack: problem: lots of our discussions and decisions get lost in Slack, no transparency
- 🅰️  Discussion: should we log the clojurebridge-orga channel publicly? diverse opinions on that.
- 🅰️ For now: We need to put the result of our discussions in GitHub issues.
- 🅰️ Discuss use of Slack, GitHub and Trello etc.
- 🅰️ We want to have a buddy system, 2 people team up for a task, worked well this time with attendee care
- 🅰️ We should have seperate channels in Slack for separate topics, maybe starting with food and attendee-care 
- 🅰️ We want to have a **Hackday** 🎉 💻  for improving our processes including:
  - a timeline
  - documentation: how to document? how to find and use existing documentation? report back and help the board
  - create a sign up app, spreadsheets is madness
### Speakers
- diverse and all women set was good
- Johannes talk was missed, maybe we can team up with him in case he cant be at the workshop
- speaker preparation has to be better and personal, mail is not enough
- 🅰️ start a mentoring system like eurucamp, brief mentors
### Code of Conduct
- We use the berlincodeofconduct.org and had a last minute briefing on how to handle it with a [wiki entry](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports) from geekfeminism wiki.
- we had 3 incidents
- documenting the incidents was good
- 🅰️  we still need to figure out the best process for us
- 🅰️ we need a accessible list of emergency numbers, such as firefighters, police etc and venue related numbers
- 🅰️ it would be good to have a separate organizer room 
### Learners group
- We want to have an intro to programming or welcome project at the learners group so people know what to do
- we need a kick off
- we need an after workshop curriculum
- we want people to come back: how?
- 🅰️ Discussion: Which date to set for the first learners group after the workshop? How to make people commit to the group?
### Curriculum
- All discussion moved to a seperate meeting
### Shirts
- It would be nice but it is a hassle with the sizes, print, material, costs, etc.
- 🅰️ Discuss it another time
### Date for the next workshop
- Kick off for orga: 15.09.2016
- Workshop: 25.11. - 26.11/2016 
- 🅰️ Discussion: should we have a winter and a summer team?
### Things we need to do for the last workshop
- Mails:
  - [x] Goddies to former attendees (Euroclojure, Lambda Island, Purely functional TV) ( ---> https://github.com/clojurebridge-berlin/organization/wiki/%5Bemail%5D-Post-workshop-goodies)
  - [x] Reminder for the project group ( ---> https://github.com/clojurebridge-berlin/organization/wiki/%5Bemail%5D-Post-workshop-goodies)
  - [x] Sponsor mail  (---> https://github.com/clojurebridge-berlin/organization/wiki/%5Bemail%5D-Sponsor-Email-Thank-You-(Version-2) ) 
- [x] Receipts: Hand them to Arne/Travis Foundation
- [ ] Delete all private data from our spreadsheets
- [x] Picture of all organizers (or drawing ✅ )
- [ ] Celebrate our success at the Ethiopian place

**Comment by vsmart** [comment]
One point to add:  

## Volunteers
- Having a set list of people for each job worked well
- Maybe some more instructions are needed + badges to mark them as volunteers
- We should have a process for having people sign up though, including CoC agreement (especially those who would otherwise not attend) 
- 🅰️ Next time, maybe someone can coordinate volunteers from the start

**Comment by Malwine** [comment]
Thanks @vsmart for preparing and sending all the mails! ❤️

**Comment by socksy** [comment]
TODO:
- [x] Add  "visualize role on the badge: coach, attendee, volunteer" to checklist wiki page
- [x] Add "have a list of who is inside the building for emergency" to checklist wiki page
- [x] Add ensuring an accessible quiet room to the checklist wiki page
- [x] Find link for checklist wiki page to make this todo list more actionable
- [ ] Set date for learner group retro/meeting (with existing coaches etc)
- [x] Set up new Slack channels for e.g. food
- [x] [Make issue for] Write up guest policy. Possibly add to website
- [x] [Make issue for] **Delete private data of last workshops** (if not done already)
- [x] [Make issue for] Prepare list of emergency contacts for Wunderlist
- [ ] [Make issue for] Internal document with contact info of all people at the workshop (and on-call person)
- [x] Write comment on speaker issue about speaker mentoring process

I think after all these are done we can close this issue.

**Comment by vsmart** [comment]
@socksy do you mean that "less-obvious"-conference-organising checklist I referenced or "our own" CljBridge checklist?  
The less-obvious-list is here: [erikr/lessobviouschecklist](https://github.com/erikr/lessobviouschecklist) and I added some things we learned, including e.g. accessibility of quiet rooms: [view PR](https://github.com/erikr/lessobviouschecklist/pull/9).  
The ClojureBridge-checklist is [here](https://github.com/clojurebridge-berlin/organization/wiki/Checklist:-Jobs-on-the-Day-%E2%9C%85) and I've added the things you listed here: [view diff](https://github.com/clojurebridge-berlin/organization/wiki/Checklist:-Jobs-on-the-Day-%E2%9C%85/_compare/49d0b93f1affd4622d6f01388922c6d897848a3f...cfb5b4f78d710a11d4eb741ef1a48fe226665597).

I checked off these task in your list above.

**Comment by vsmart** [comment]
I checked off these: 
- Set up new Slack channels for e.g. food
  ==> Has been done, e.g. _catering_
- [Make issue for] Write up guest policy. Possibly add to website
  ==> https://github.com/clojurebridge-berlin/organization/issues/123
- [Make issue for] Delete private data of last workshops (if not done already)
  ==> In progress. (See Slack) 
- [Make issue for] Prepare list of emergency contacts for Wunderlist
  ==> I put this on the venue-jobs checklist. https://github.com/clojurebridge-berlin/organization/wiki/Checklist:-Jobs-on-the-Day-%E2%9C%85
  It doesn't need its own issue as there is a list already in the venue (just need to bring it)

**Comment by vsmart** [comment]
Still remaining is: 
- Set date for learner group retro/meeting (with existing coaches etc)

We haven't done this yet but I think there won't be time before the workshop. However, we've discussed [How to make more people come to the learning group](https://github.com/clojurebridge-berlin/organization/issues/113) and as a result of this we've decided to set up a [curriculum for the 1st month](https://github.com/clojurebridge-berlin/organization/issues/118) and a mailing list. 
I think we can consider this as taken care of for now 
- [Make issue for] Internal document with contact info of all people at the workshop (and on-call person)

I wasn't sure about this one. Is this for orga people only or attendees? We have the attendee check-in list and the who-is-in-the-building-list. 

Can we close this issue? Thumbs-up please :D

**Comment by plexus** [comment]
Thanks @vsmart.

I think there are two things regarding people lists, we should have an idea of everyone who's there, attendees, coaches, orga, visitors, volunteers, caterer, etc. And then we should have a contact list with police, fire dept, ambulance. That last one is probably already somewhere there in the office, we just need to make sure we are aware of where it is.

But I'd say we're good here, happy to hit the button for you ✅

**Comment by vsmart** [comment]
@plexus Yay!!! Thank you! 👏 

![dance gif](http://i.giphy.com/26BRwEbGXxwKUu1fa.gif)

**Comment by plexus** [comment]
That cat is so not amused 😂
